docs/security/email-link-protection.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Email Link Protection
+description: Protect email link sign-ins and sign-ups
+---
+
+# Email Link Protection
+
+Clerk provides an additional setting to improve security for email link sign-ins and sign-ups. When this setting is enabled, only email links opened on the same device and browser that a sign-in or sign-up was initiated on will successfully authenticate the user.
+
+Enabling this protection means that the following behavior cannot happen:
+
+* A malicious actor knows the email of a user and requests an email link to sign-in or sign-up.
+* The user opens the email and clicks the link without thinking, or the user's email provider follows links in emails for phishing or spam protection.
+* The malicious actor is able to gain access to the user's account because the email link as been opened successfully.
```

Comment:
```suggestion
1. A malicious actor knows the email of a user and requests an email link to sign-in or sign-up.
1. The user opens the email and visits the link, or the user's email provider follows links in emails for phishing or spam protection.
1. The malicious actor is able to gain access to the user's account because the email link as been opened successfully.
```
Changed to an ordered list since it's a sequence of events, and changed "click" to "visit", as "click" is less accessible language